integration/typescript/runner.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
Yes and no. I opted to use bash instead as gulp pipelines discourage the use of "temp" directories which is what makes this flow so consistent. I could do that in gulp but it would be going against a best practice. 

In addition, not everyone will be consuming the SDK via gulp and not all of the scripts will be possible in gulp (e.g. reactnative/cordova testing which is soon to come).